src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -25,6 +25,7 @@
 
 package javax.swing.text.html;
 
+import sun.awt.AppContext;
```

Comment:
In JDK code, internal classes are listed below public `java.*` and `javax.*` packages. The order was correct.

Reply:
Thanks, I fixed this in my IntelliJ auto import settings.